marathon.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "hdfs",
+    "cmd": "cd hdfs* && ./bin/hdfs-mesos",
+    "uris": [
+        ""
```

Comment:
Could you add the docker file here to take care of this TODO? Since I have given you access to dockerhub, you should be able to upload it now.